web/build.sh

```diff
@@ -69,6 +68,14 @@ if builder_start_action clean; then
   builder_finish_action success clean
 fi
 
+# Do not call child actions for configure - they all do the same thing, and it can take a while.
+
+if builder_start_action configure; then
+  verify_npm_setup
+
+  builder_finish_action success configure
+fi
```

Comment:
This change should not be necessary if each of the child actions uses `/node_modules` as their described output?

Reply:
Ah, I remember what prompted it now.  `--force-deps` builds will ignore that optimization.

Reply:
Okay, go with this for now, with a longer term solution described in #8471.

Reply:
I've changed the base for this to a PR merging a post-#8471 `master` into the feature branch so that I can pre-emptively fix this properly.